otter/auth.py

```diff
@@ -235,6 +235,15 @@ def __init__(self, identity_admin_user, identity_admin_password, url, admin_url)
 
     @wait(ignore_kwargs=['log'])
     def _auth_me(self, log=None):
+        def _log_failed_auth(err):
+            """
+            Log this as a string we know we can find in the logging feed
+            """
+            if log:
+                log.err(err, 'Failed to get a new identity admin token.',
+                        otter_msg_type='impersonation-failed')
```

Comment:
I hate to keep nitpicking (especially since it's nitpicking *my own* suggestion), but now that I saw your new message here and read the surrounding code, I realize that this isn't an error in the impersonation step, but just the admin login step, so I guess the otter_msg_type should be 'admin-login-failed'. If you wanna make this change you can go ahead and merge right after that. Sorry & thanks. :-)